src/visualizers/Chaos.ts

```diff
@@ -189,52 +199,44 @@ class Chaos extends P5Visualizer {
     // colour palette
     private currentPalette = new Palette()
 
-    checkParameters() {
-        const status = super.checkParameters()
+    checkParameters(params: {[key: string]: unknown}) {
+        const status = super.checkParameters(params)
```

Comment:
similar typing issues here as with sequences, I won't put comments in every instance.